pkg/kubectl/cmd/create_deamonset.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+var (
+	daemonsetLong = templates.LongDesc(i18n.T(`
+	Create a daemonset with the specified name.`))
```

Comment:
Please provide a better description of daemonsets here, including what they do.  This should be educational to new users who know nothing about kubernetes.  I.e. try to answer:

1. what is a daemonset
2. what are the key use cases and how will this command help me start one
3. where to go to find more information about daemonsets

These create commands help users bridge the gap from novice to productive.